examples/default.yaml

```diff
@@ -229,13 +229,21 @@ firmware:
   legacyBIOS: null
 
 video:
-  # QEMU display, e.g., "none", "cocoa", "sdl", "gtk", "default".
+  # QEMU display, e.g., "none", "cocoa", "sdl", "gtk", "vnc", "default".
   # Choosing "none" will hide the video output, and not show any window.
+  # Choosing "vnc" will use a network server, and not show any window.
   # Choosing "default" will pick the first available of: gtk, sdl, cocoa.
-  # As of QEMU v6.2, enabling this is known to have negative impact
+  # As of QEMU v6.2, enabling anything but none or vnc is known to have negative impact
   # on performance on macOS hosts: https://gitlab.com/qemu-project/qemu/-/issues/334
   # 🟢 Builtin default: "none"
   display: null
+  # VNC (Virtual Network Computing) is a platform-independent graphical
+  # desktop-sharing system that uses the Remote Frame Buffer protocol (RFB)
```

Comment:
Let's mark this experimental.

Reply:
@afbjorklund @jandubois WDYT?

Reply:
FYI: The experimental features are now tracked in https://github.com/lima-vm/lima/blob/master/docs/experimental.md

See also https://github.com/lima-vm/lima/search?q=experimental